### ISSUE TITLE
Missing softmax in BertForQuestionAnswering after linear layer?

### ISSUE THREAD

**Issue by tsaikenyoo** [huggingface/pytorch-pretrained-BERT#209]
https://github.com/huggingface/pytorch-pretrained-BERT/blob/0a9d7c7edb20a3e82cfbb4b72515575543784823/pytorch_pretrained_bert/modeling.py#L1089-L1113

It seems there should be a softmax after the linear layer, or did I miss something?

**Comment by rodgzilla** [comment]
It depends on what you use as loss, as mentioned in the [documentation](https://pytorch.org/docs/stable/nn.html#crossentropyloss):

>This criterion combines nn.LogSoftmax() and nn.NLLLoss() in one single class.
